sql/analyzer/resolve_column_defaults.go

```diff
@@ -413,3 +425,27 @@ func backtickDefault(wrap *expression.Wrapper) (sql.Expression, transform.TreeId
 	nd.Expr = newExpr
 	return expression.WrapExpression(&nd), transform.NewTree, nil
 }
+
+// normalizeDefault ensures that default values that are literals are normalized string literals. This is necessary for
```

Comment:
So if I'm understanding this PR correctly, we're storing the normalized default as the column type, not a string literal, and it gets converted to string literal when we call `show create`. If so, we should update this comment.